store/jsondb/jsondb.go

```diff
@@ -212,9 +212,11 @@ func (o *JsonDB) GetClientByID(clientID string, qrCodeSettings model.QRCodeSetti
 		if !qrCodeSettings.IncludeMTU {
 			globalSettings.MTU = 0
 		}
+		/*
 		if !qrCodeSettings.IncludeFwMark {
-			globalSettings.ForwardMark = ""
+			globalSettings.FirewallMark = ""
 		}
+		*/
```

Comment:
Maybe we can just remove all commented codes if we don't need them. The same for other places